counsel.el

```diff
@@ -61,37 +61,19 @@ N is obtained from `counsel-more-chars-alist'."
       (list "" (format "%d chars more" diff)))))
 
 (defun counsel-unquote-regex-parens (str)
-  "Unquote regex parenthesis in STR."
```

Comment:
Would be nice to add some tests for this.

Reply:
I wrote this commit a few moons ago, and made sure to test that my refactor was equivalent at the time, but I stupidly didn't commit the tests. Sorry, will do so in a follow-up PR.

Reply:
Done: #1708.